spec/lib/environment_config_transmogrifier_spec.rb

```diff
@@ -63,6 +64,31 @@
       expect(new_config['properties']['parent_key']['child_key']['grandchild_key']).to eq 'bar'
     end
 
+    it 'should error on triple-paren' do
+      # Arrange
+      environment_templates = {
+        'properties.parent_key.child_key.grandchild_key' => 'f(((MY_FOO_VAR)))'
+      }
+      # Act
+      # Assert
+      expect {
+        EnvironmentConfigTransmogrifier.transmogrify(@base_config, environment_templates)
+      }.to(raise_exception(LoadYamlFromMustacheError,
+                           /Could not load config key.*Illegal content in tag/))
+    end
+
+    it 'should support changing templates inline' do
+      # Arrange
+      environment_templates = {
+        'properties.parent_key.child_key.grandchild_key' => '((={{ }}=))f({{MY_FOO_VAR}})'
```

Comment:
I wrote a test that tries that, and get this error message:
```
     LoadYamlFromMustacheError:
       Could not load config key 'properties.parent_key.child_key.grandchild_key': Illegal content in tag: Error at or near position 17 of template value. (Illegal content in tag
         Line 1
           {{=(( ))=}}f(("("))((MY_FOO_VAR))((")"))
```

And that terminology is the mustache way of setting delimiters, so I don't see the problem